docs/source/ecosystem/pytorch_ecoystem.rst

```diff
@@ -1,4 +1,4 @@
-Pytorch Ecosystem Examples
+PyTorch Ecosystem Examples
 ==========================
 
-- `Pytorch Geometric: Deep learning on Graphs and other irregular structures <https://github.com/PyTorchLightning/pytorch-lightning/tree/master/pl_examples/pytorch_ecosystem/pytorch_geometric/README.md>`_.
+- `PyTorch Geometric: Deep learning on graphs and other irregular structures <https://github.com/rusty1s/pytorch_geometric/tree/master/examples/pytorch_lightning>`_.
```

Comment:
@tchaton check is this good?
We also have Geometric in the community samples, should we merge these sections into one page?

Reply:
Yes, just pushed to your branch.

Reply:
thanks